testing/cli-e2etest/testscenarios/environment/list_environments_test.go

```diff
@@ -127,12 +127,13 @@ func TestListEnvironments(t *testing.T) {
 		result := tracetestcli.Exec(t, "list environment --sortBy name --sortDirection asc --output json", tracetestcli.WithCLIConfig(cliConfig))
 		helpers.RequireExitCodeEqual(t, result, 0)
 
-		environmentVarsList := helpers.UnmarshalJSON[[]types.EnvironmentResource](t, result.StdOut)
-		require.Len(environmentVarsList, 3)
+		environmentVarsList := helpers.UnmarshalJSON[types.ResourceList[types.EnvironmentResource]](t, result.StdOut)
```

Comment:
this scenario was incorrect (as well as the old implementation). it was returning an array of resources instead of a `ResourceList`